src/test/java/com/cloudbees/jenkins/support/impl/AbstractItemComponentTest.java

```diff
@@ -148,11 +147,10 @@ public void addContentsFromFreestyle() throws Exception {
     public void addContentsFromPipeline() throws Exception {
         MockFolder folder = j.createFolder(FOLDER_NAME);
         WorkflowJob p = folder.createProject(WorkflowJob.class, JOB_NAME);
-        p.setDefinition(new CpsFlowDefinition("node {semaphore 'wait'}", true));
+        p.setDefinition(new CpsFlowDefinition("node { echo 'test' }", true));
         WorkflowRun workflowRun = Optional.ofNullable(p.scheduleBuild2(0))
                 .orElseThrow(AssertionFailedError::new)
                 .waitForStart();
-        SemaphoreStep.success("wait/1", workflowRun);
```

Comment:
If you suggest in the fix to only use `SemaphoreStep.success("wait/1", null);` here instead of replacing it with `echo 'test'` in the pipeline then let me know.